Source/WebKit/Platform/spi/ios/UIKitSPI.h

```diff
@@ -1542,6 +1546,12 @@ typedef NS_ENUM(NSUInteger, _UIContextMenuLayout) {
 @property (nonatomic, setter=_setSuppressSoftwareKeyboard:) BOOL _suppressSoftwareKeyboard;
 @end
 
+#if HAVE(PASTEBOARD_DATA_OWNER) || HAVE(LINK_PREVIEW)
+@interface UIPasteboard ()
++ (void)_performAsDataOwner:(_UIDataOwner)dataOwner block:(void(^)(void))block;
+@end
+#endif
```

Comment:
This doesn't need to be guarded (it's available on all iOS versions we support), and HAVE(PASTEBOARD_DATA_OWNER) was cleaned up in #3726.